packages/react/src/SelectPanel/SelectPanel.tsx

```diff
@@ -448,167 +398,172 @@ export function SelectPanel({
     error: <StopIcon size={16} />,
   }
 
+  function getMessage() {
+    if (items.length === 0 && !message) {
+      return DefaultEmptyMessage
+    } else if (message) {
+      return (
+        <SelectPanelMessage title={message.title} variant={message.variant}>
+          {message.body}
+        </SelectPanelMessage>
+      )
+    }
+  }
+
   return (
-    <LiveRegion>
-      <AnchoredOverlay
-        renderAnchor={renderMenuAnchor}
-        anchorRef={anchorRef}
-        open={open}
-        onOpen={onOpen}
-        onClose={onClose}
-        overlayProps={{
-          role: 'dialog',
-          'aria-labelledby': titleId,
-          'aria-describedby': subtitle ? subtitleId : undefined,
-          ...overlayProps,
-        }}
-        focusTrapSettings={focusTrapSettings}
-        focusZoneSettings={focusZoneSettings}
-        height={height}
-        width={width}
-        anchorId={id}
-        variant={usingFullScreenOnNarrow ? {regular: 'anchored', narrow: 'fullscreen'} : undefined}
-        pinPosition={!height}
+    <AnchoredOverlay
+      renderAnchor={renderMenuAnchor}
+      anchorRef={anchorRef}
+      open={open}
+      onOpen={onOpen}
+      onClose={onClose}
+      overlayProps={{
+        role: 'dialog',
+        'aria-labelledby': titleId,
+        'aria-describedby': subtitle ? subtitleId : undefined,
+        ...overlayProps,
+        style: {
+          '--max-height': overlayProps?.maxHeight ? heightMap[overlayProps.maxHeight] : heightMap['large'],
+        } as React.CSSProperties,
+      }}
+      focusTrapSettings={focusTrapSettings}
+      focusZoneSettings={focusZoneSettings}
+      height={height}
+      width={width}
+      anchorId={id}
+      variant={usingFullScreenOnNarrow ? {regular: 'anchored', narrow: 'fullscreen'} : undefined}
+      pinPosition={!height}
+      className={classes.Overlay}
+    >
+      <Box
+        sx={enabled ? undefined : {display: 'flex', flexDirection: 'column', height: 'inherit', maxHeight: 'inherit'}}
+        className={enabled ? classes.Wrapper : undefined}
       >
-        <LiveRegionOutlet />
-        {usingModernActionList ? null : (
-          <Message
-            value={
-              filterValue === ''
-                ? 'Showing all items'
-                : items.length <= 0
-                  ? 'No matching items'
-                  : `${items.length} matching ${items.length === 1 ? 'item' : 'items'}`
-            }
-          />
```

Comment:
Removing this in favor of the regular announcements, not sure why he had this on top of the announcements, I tested without it and all looks good, seems to me that we were over-announcing on deprecated